narwhal/worker/src/worker.rs

```diff
@@ -235,6 +235,8 @@ impl Worker {
             config.inbound_request_timeout_ms = Some(300_000);
             config.outbound_request_timeout_ms = Some(300_000);
             config.shutdown_idle_timeout_ms = Some(1_000);
+            config.connectivity_check_interval_ms = Some(2_000);
+            config.connection_backoff_ms = Some(5_000);
```

Comment:
Are you still happy with the default maximum backoff value https://mystenlabs.github.io/anemo/anemo/struct.Config.html#structfield.max_connection_backoff_ms?

Reply:
Good point! Probably something lower, like 20 seconds, might be a bit better. I'll amend the PR.

Reply:
Just know that in the event that a node is down, we're going to have N validators trying to connect to a dead node roughly every ~20 seconds.

Reply:
Yes, I would say at least for the current setup (40 validators) this will probably be ok - maybe once we deal with larger setup we could potentially increase this number a little bit, but still I wouldn't go really high.

Reply:
> Just know that in the event that a node is down, we're going to have N validators trying to connect to a dead node roughly every ~20 seconds.

Is the worry that the dead node might not actually be dead, but we end up killing it with retries anyway? Even a couple hundred validators once every 20s doesn't seem like it should be a problem.